api/geth_backend.go

```diff
@@ -124,6 +127,7 @@ func (b *GethStatusBackend) initialize() {
 	b.log = log.New("package", "status-go/api.GethStatusBackend")
 	b.LocalPairingStateManager = new(statecontrol.ProcessStateManager)
 	b.LocalPairingStateManager.SetPairing(false)
+
```

Comment:
lol adding an empty line is a smart way of increasing the test coverage 😄 
<img width="127" alt="image" src="https://github.com/status-im/status-go/assets/25482501/566e370d-35af-4808-9870-63f0943a24e0">